pkg/nuctl/command/export.go

```diff
@@ -135,9 +143,15 @@ func newExportProjectCommandeer(exportCommandeer *exportCommandeer) *exportProje
 	}
 
 	cmd := &cobra.Command{
-		Use:     "projects [name]",
-		Aliases: []string{"proj", "prj", "project"},
-		Short:   "(or project) Export project with all it's functions and function events",
+		Use:     "projects [<project>]",
+		Aliases: []string{"project", "prj", "proj"},
+        Short:   "(or project) Export projects",
+		Long:    "(or project) Export the configuration of a specific project (incuding all
```

Comment:
```suggestion
		Use:     "projects [<project>]",
		Aliases: []string{"project", "prj", "proj"},
		Short:   "(or project) Export projects",
		Long:    "(or project) Export the configuration of a specific project (incuding all
```
Inconsistent indentation here as well, please align the `Short` member to the rest of the members - lint issue.